deep-learning/_posts/2017-03-01-alexnet.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: review
 title: (AlexNet) ImageNet Classification with Deep Convolutional Neural Networks
```

Comment:
I think that "AlexNet : ImageNet Classification..." would be more readable than using parentheses. If you use a colon you must surround the title with quotes. (not a big deal)